bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```diff
@@ -357,8 +357,8 @@ allowFetchAllRecordings=true
 # The directory where the pre-built configs are stored
 configDir=/var/bigbluebutton/configs
 
-# Disable this option to avoid overriding client settings through /create call
-allowOverrideClientSettingsOnCreateCall=true
+# Enable this option to avoid overriding client settings through /create call
```

Comment:
Enable this option to avoid overriding -> Enable this option to **allow** overriding